user-service/src/main/java/com/ecommercepractice/userservice/models/User.java

```diff
@@ -0,0 +1,13 @@
+package com.ecommercepractice.userservice.models;
+
+import lombok.*;
+
+@Getter @Setter @NoArgsConstructor @AllArgsConstructor @ToString
+public class User {
+    private  String user_id;
```

Comment:
use camel case format on all the attributes

Reply:
Ok